Include/arm_nnsupportfunctions.h

```diff
@@ -920,6 +920,44 @@ __STATIC_FORCEINLINE const int8_t *read_and_pad(const int8_t *source, int32_t *o
     return source;
 }
 
+/**
+ * @brief read and expand one s8 word into two s16 words with ordering and addition.
+ */
+__STATIC_FORCEINLINE void read_pad_and_add(const int8_t *source, int32_t *out1, int32_t *out2, const uint32_t add)
```

Comment:
the current name aligns with the non-addition version of this function, so if we rename this one it would make sense to rename the others, but that would break existing abi

Reply:
Since these are support functions, I don't see any harm in renaming them

Reply:
renamed